src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getRoleByName(String roleName);
```

Comment:
```suggestion
    Role getByName(String roleName);
```

Reply:
I've changed it, somehow it didn't go through.